tests/browserstack/specs/ballotMainTest.js

```diff
@@ -214,18 +263,18 @@ describe('Basic cross-platform We Vote test',  () => {
     // }, stickyNoButton); // Click on Sticky Header No
 
     // Go back to ballot
-    await simpleClick('backToLinkTabHeader');
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
+    // await simpleClick('backToLinkTabHeader');
```

Comment:
Instead of commenting out the rest of the lines, you can use:
```
      // Stop the script here for now
      assert(true);
      driver.quit();
```